### ISSUE TITLE
Testing pointpillars pre-trained on kitti dataset on nuscenes data

### ISSUE THREAD

**Issue by mohamed11981198** [open-mmlab/mmdetection3d#512]
How can i test pointpillars pre trained on kitti (64 layer lidar )on different lidar data like 32 layer lidar of nuscenes dataset or custom 128  layers lidar data

**Comment by Tai-Wang** [comment]
Please refer to the [documentation](https://mmdetection3d.readthedocs.io/en/latest/0_demo.html) for demo. You can basically get a sample data from nuScenes and follow the way given in the demo doc.

**Comment by mohamed11981198** [comment]
I have tried the documentation demo but the problem in the nuscenes point cloud size not match with the kitti pretrained model expected input i tried to upsample  the point cloud by padding and the demo run but the detector did not work correctly

**Comment by Wuziyi616** [comment]
What do you mean by "did not work correctly"? BTW, the generalization of 3d point cloud detector is not very good, especially when the property of input such as point density changes. It's reasonable if you get bad prediction results when testing 64 layer trained model on 32 layer data.

**Comment by mohamed11981198** [comment]
Yes ,the model predict a random and and not correct bounding boxes , i will try to fine tune the model with 32 layer lidar data first,thank you very much

**Comment by Tai-Wang** [comment]
There are many underlying problems when you decide to transfer a pretrained model to a new dataset. Apart from different lines of LiDAR, the intensity of point cloud data on nuScenes is also very different from that on KITTI. Its fourth dimension corresponds to timestamp currently, so this can be one of the important reasons that make the adaptation further worse.

**Comment by Wuziyi616** [comment]
Feel free to reopen this issue if you have any further questions :)
